app/assets/js/components/BatchEdit/ModalRemove.js

```diff
@@ -50,6 +50,8 @@ export default function BatchEditAboutModalRemove({
   useEffect(() => {
     if (items.length > 0) {
       setCandidateList(setupCandidateList(items));
+    } else {
+      setCandidateList([]);
```

Comment:
This resolves the issue, and makes me wonder how this could have worked previously.